cl/scrapers/management/commands/cl_scrape_opinions.py

```diff
@@ -230,6 +230,18 @@ def add_arguments(self, parser):
             default=False,
             help="Disable duplicate aborting.",
         )
+        parser.add_argument(
```

Comment:
I think this arguments should be added in the backscraper command to keep the scraper separate from the backscraper. With something like:

```
    def add_arguments(self, parser):
        super().add_arguments(parser)
        parser.add_argument(
            "--backscrape-start",
            dest="backscrape_start",
            help="Starting value for backscraper iterable creation. "
                 "Each scraper handles the parsing of the argument,"
                 "since the value may represent a year, a string, a date, etc.",
        )
        parser.add_argument(
            "--backscrape-end",
            dest="backscrape_end",
            help="End value for backscraper iterable creation.",
        )
```